subjects/sortedlistmerge/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"piscine"
+)
+
+type NodeI struct {
```

Comment:
To define their function it's better that the students define this structure in their solution. If that's not the case, every time there's a `piscine.NodeI` here should be changed to `NodeI`